docs/content/Examples-Tutorials-Recipes/Pagination.md

```diff
@@ -0,0 +1,132 @@
+---
+title: Pagination
+permalink: /recipes/pagination
+category: Examples & Tutorials
+subCategory: Queries
+menuOrder: 2
+---
+
+## Use case
+
+We want to display a table of data with hundreds of rows. To make the table
+easier to digest and to improve the performance of the query, we'll use
+pagination. With the recipe below, we'll get the orders list sorted by the order
+number. Every page will have 5 orders.
+
+## Data schema
+
+We have the following data schema.
+
+```javascript
+cube(`Orders`, {
+  sql: `SELECT * FROM public.orders`,
+
+  measures: {
+    count: {
+      type: `count`,
+    },
+  },
+
+  dimensions: {
+    number: {
+      sql: `number`,
+      type: `number`,
+    },
+
+    createdAt: {
+      sql: `created_at`,
+      type: `time`,
+    },
+  },
+});
+```
+
+## Query
+
+To creating the pagination we can use the `limit` and `offset` query properties.
+First, we get the number of all orders that we have. Then we should set the
+`limit` and `offset` properties for the queries that will get the orders from
+the Cube.js API.
+
+```bash
+// Get count of the orders
+curl cube:4000/cubejs-api/v1/load \
+-G -s --data-urlencode "query={"measures": ["Orders.count"]}"
```

Comment:
I think you can just show the queries, all this `curl...` stuff just makes it harder to understand what's going on.